specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/examples/compute/GetACapacityReservation.json

```diff
@@ -0,0 +1,59 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscriptionId}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2021-07-01",
+    "capacityReservationGroupName": "myCapacityReservationGroup",
+    "capacityReservationName": "myCapacityReservation"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "reservationId": "{GUID}",
+          "provisioningTime": "2021-06-27T01:02:38.3138469+00:00",
+          "virtualMachinesAssociated": [
+            {
+              "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachines/myVM1"
+            },
+            {
+              "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachines/myVM2"
+            }
+          ],
```

Comment:
In virtualMachinesAssociated example we should show more VM resource id than the ones returned in virtualMachinesAllocated as part of instanceView.

Reply:
updated